src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -10,16 +10,35 @@
 
 namespace System.Text.Encodings.Web
 {
+    /// <summary>
+    /// Represents a type used to do HTML encoding.
+    /// </summary>
     public abstract class HtmlEncoder : TextEncoder
     {
+        /// <summary>
+        /// Returns default built-in instance of HtmlEncoder.
+        /// </summary>
         public static HtmlEncoder Default
         {
             get { return DefaultHtmlEncoder.Singleton; }
         }
+
+        /// <summary>
+        /// Creates a new instalce of HtmlEncoder with provided settings.
```

Comment:
Nit: throughout you can also use XML like ```<see cref="HtmlEncode" />``` to enable tools to understand that you're referring to a class, provide appropriate hyperlinks, etc.